docs/usage.md

```diff
@@ -4,54 +4,215 @@
 
 > _Documentation of pipeline parameters is generated automatically from the pipeline schema and can no longer be found in markdown files._
 
-## Introduction
-
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
-
-## Running the pipeline
+## RUNNING THE PIPELINE
 
 The typical command for running the pipeline is as follows:
 
-```bash
-nextflow run nf-core/rnavar --input '*_R{1,2}.fastq.gz' -profile docker
+```console
+nextflow run nf-core/rnavar --input samplesheet.csv --genome GRCh38 -profile docker
```

Comment:
--outdir is mandatory now, or did that change again?

Reply:
not mandatory now, as it defaults to `./results`

Reply:
Added in the command line.